daemon/src/wallet_sync.rs

```diff
@@ -0,0 +1,24 @@
+use crate::wallet::Wallet;
```

Comment:
nit: rename file to `wallet_sync_actor.rs`?

Reply:
I don't think this is an actor

Reply:
fair enough, but it's a long-running task, and it is a bit misleading when it just starts with `new()` - maybe it could start with `spawn()` or something like that? 

or can we reason about "actors" without inboxes?

Reply:
I think that is the main thing right there, if something doesn't have an inbox, it is not an actor :)